app/models/vm_or_template.rb

```diff
@@ -1884,6 +1884,10 @@ def supports_snapshots?
     false
   end
 
+  def supports_provisioning?
+    true
```

Comment:
@h-kataria I really do not think this should have been merged but please address these issues in a followup PR.